firefox-ios/Client/Frontend/Home/Homepage Rebuild/ContextMenu/ContextMenuCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+
+protocol ContextMenuCoordinatorDelegate: AnyObject {
+    func dismissFlow()
+}
+
+final class ContextMenuCoordinator: BaseCoordinator, ContextMenuCoordinatorDelegate {
+    weak var parentCoordinator: ParentCoordinatorDelegate?
+    private let windowUUID: WindowUUID
+    private let configuration: ContextMenuConfiguration
+
+    init(
+        configuration: ContextMenuConfiguration,
+        router: Router,
+        windowUUID: WindowUUID
+    ) {
+        self.configuration = configuration
+        self.windowUUID = windowUUID
+        super.init(router: router)
+    }
+
+    func start() {
```

Comment:
nit: I know that context menu coordinator is indicative of what this class does but is there potentially better naming for the function `start` that might give more clarity to what is actually happening here?

Reply:
hmm, this is following the same pattern as other coordinators, so maybe if we want to give clarity, we should update coordinators in general to give consistency of the method name? Going to merge in this PR and can follow up on this separately.